docs/examples/builds/multi-root-editor.md

```diff
@@ -0,0 +1,187 @@
+				toolbarContainer.classList.add( 'sticky' );
+			} else {
+				toolbarContainer.classList.remove( 'sticky' );
+			}
+		} );
+	} )
+	.catch( error => {
+		console.error( 'There was a problem initializing the editor.', error );
+	} );
+
+```
+
+</details>
+
+<details>
+<summary>View editor content listing</summary>
+
+```html
+<div id="toolbar"></div>
+<!--
+	Wrapping the structure inside a pair of
+	contenteditable="true" + contenteditable="false" elements
+	is required to provide proper caret handling when
+	using arrow keys at the start and end of an editable area.
+	
+	You can skip them if you don't want to move the
+	caret between editable areas using arrow keys.
+!--> 
+<div contenteditable="true">
+	<div contenteditable="false">
+		<div id="header">
+			Header content is inserted here.
+		</div>
+		<div id="content">
+			Main content is inserted here.
+		</div>
+		<div class="boxes">
+			<div class="box box-left editor">
+				<div id="left-side">
+					Left-side box content is inserted here.
+				</div>
+			</div>
+			<div class="box box-right editor">
+				<div id="right-side">
+					Right-side box content is inserted here.
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+
+<style>
+	.box {
+		width: 50%;
+	}
+
+	.box-left {
+		margin-right: 10px;
+	}
+
+	/*
+		Make the editable "fill" the whole box.
+		The box will grow if the other box grows too.
+		This makes the whole box "clickable".
+	 */
+	.box .ck-editor__editable {
+		height: 100%;
+	}
+
+	/*
+		When toolbar receives this class, it becomes sticky.
+		If the toolbar would be scrolled outside of the visible area,
+		instead it is kept at the top edge of the window.
+	 */
+	#toolbar.sticky {
+		position: sticky;
+		top: 0px;
+		z-index: 10;
+	}
+</style>
+```
+
+</details>
+
+## Setting and reading editor data
+
+Please note that setting and reading the editor data is different for multi-root editor.
+
+<details>
+<summary>Pass an object when setting the editor data</summary>
+
+Setting the data using `editor.setData()`:
+```js
+	editor.setData( {
+		header: '<p>Content for header part.</p>',
+		content: '<p>Content for main part.</p>',
+		leftSide: '<p>Content for left-side box.</p>',
+		rightSide: '<p>Content for right-side box.</p>'
+	} );
+```
+
+Setting the data through `config.initialData`:
+```js
+	MultiRootEditor.create(
+		{
+			header: document.querySelector( '#header' ),
+			content: document.querySelector( '#content' ),
+			leftSide: document.querySelector( '#left-side' ),
+			rightSide: document.querySelector( '#right-side' )
+		},
+		{
+			initialData: {
+				header: '<p>Content for header part.</p>',
+				content: '<p>Content for main part.</p>',
+				leftSide: '<p>Content for left-side box.</p>',
+				rightSide: '<p>Content for right-side box.</p>'
+			}
+		}
+	);
+```
+</details>
+
+<details>
+<summary>Specify root name when obtaining the data</summary>
+
+```js
+	editor.getData( { rootName: 'leftSide' } ); // -> '<p>Content for left-side box.</p>'
```

Comment:
As you can see I decided not to change the API for getting the data at this point.